modules/charon-core/src/test/java/org/wso2/charon3/core/protocol/endpoints/MeResourceManagerTest.java

```diff
@@ -0,0 +1,1547 @@
+import static org.mockito.Matchers.anyObject;
+import static org.mockito.Mockito.doThrow;
+import static org.powermock.api.mockito.PowerMockito.mock;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+/**
+ * Test class of MeResourceManager.
+ */
+@PrepareForTest({AbstractResourceManager.class})
+public class MeResourceManagerTest extends PowerMockTestCase {
+
+    private static final String NEW_USER_SCIM_OBJECT_STRING = "{\n" +
+            "  \"schemas\": \n" +
+            "    [\"urn:ietf:params:scim:schemas:core:2.0:User\",\n" +
+            "    \"urn:ietf:params:scim:schemas:extension:enterprise:2.0:User\"\n" +
+            "    ],\n" +
+            "\"meta\": {\n" +
+            "    \"created\": \"2018-08-17T10:34:29Z\",\n" +
+            "    \"location\": \"ENDPOINT/008bba85-451d-414b-87de-c03b5a1f4217\",\n" +
+            "    \"lastModified\": \"2018-08-17T10:34:29Z\",\n" +
+            "    \"resourceType\": \"User\"\n" +
+            "},\n" +
+            " \"name\": {\n" +
+            "    \"givenName\": \"Kim\",\n" +
+            "    \"familyName\": \"Berry\",\n" +
+            "    \"formatted\": \"Kim Berry\"\n" +
+            "  },\n" +
+            "  \"userName\": \"kimjohn  \",\n" +
+            "  \"password\": \"kim123\",\n" +
+            "  \"id\": \"123\",\n" +
+            "  \"emails\": [\n" +
+            "      {\n" +
+            "        \"type\": \"home\",\n" +
+            "        \"value\": \"john@gmail.com\",\n" +
+            "         \"primary\": true\n" +
+            "      }\n" +
+            "  ]\n" +
+            "}";
+
+    private static final String NEW_USER_SCIM_OBJECT_STRING_UPDATE = "{\n" +
+            "  \"schemas\": \n" +
+            "    [\"urn:ietf:params:scim:schemas:core:2.0:User\",\n" +
+            "    \"urn:ietf:params:scim:schemas:extension:enterprise:2.0:User\"\n" +
+            "    ],\n" +
+            "\"meta\": {\n" +
+            "    \"created\": \"2018-08-17T10:34:29Z\",\n" +
+            "    \"location\": \"ENDPOINT/008bba85-451d-414b-87de-c03b5a1f4217\",\n" +
+            "    \"lastModified\": \"2018-08-17T10:34:29Z\",\n" +
+            "    \"resourceType\": \"User\"\n" +
+            "},\n" +
+            " \"name\": {\n" +
+            "    \"givenName\": \"Kim\",\n" +
+            "    \"familyName\": \"Berry\",\n" +
+            "    \"formatted\": \"Kim Berry\"\n" +
+            "  },\n" +
+            "  \"userName\": \"kimjohn  \",\n" +
+            "  \"password\": \"kim123\",\n" +
+            "  \"id\": \"123\",\n" +
+            "  \"emails\": [\n" +
+            "      {\n" +
+            "        \"type\": \"home\",\n" +
+            "        \"value\": \"john2@gmail.com\",\n" +
+            "         \"primary\": true\n" +
+            "      }\n" +
+            "  ]\n" +
+            "}";
+
+    private static final String NEWUSER_SCIM_OBJECT_STRING_FOR_PATCH = "{\n" +
```

Comment:
```suggestion
    private static final String NEW_USER_SCIM_OBJECT_STRING_FOR_PATCH = "{\n" +
```

Reply:
Fixed in commit a5699e694bc31369fb4a347d4a0a12177e73775d